packages/hackmcx-api/src/controllers/users.controller.js

```diff
@@ -22,38 +23,49 @@ export async function getUsersByUserId(req, res){
         })
 }
 
-export async function postUser(req, res){
+function userValidation(req){
     const emptyUsername = !req.body.username || /^\s*$/.test(req.body.username);
```

Comment:
I think at this point, checking for null or whitespace should be a helper function, with the regex compiled.